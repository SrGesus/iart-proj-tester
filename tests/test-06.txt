FC	FE	VC
BC	LV	BD
VE	FB	FB