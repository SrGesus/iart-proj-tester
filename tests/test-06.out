FB	FD	VE
BD	LH	BE
VD	FE	FC
